spec.bs

```diff
@@ -791,9 +805,11 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
       with |biddingSignalsUrl| and true.
     1. [=map/For each=] joiningOrigin → |groups| of |perSignalsUrlGenerator|:
       1. [=list/For each=] |ig| of |groups|:
+        1. If |ig|'s [=interest group/bidding url=] is undefined, [=iteration/continue=].
         1. TODO: If rerunning `generateBid()` with only k-anonymous ads, remember to swap out |ig|'s ads
           to just k-anonymous ones so that [=validating an ad url=] excludes the non-k-anonymous ones.
-        1. TODO: Let |interestGroup| be ... from |ig| ... minus priority and prioritySignalsOverrides and any browser-defined pieces
+        1. Let |igGenerateBid| be the result of [=building an interest group passed to generateBid=]
+          with |ig|.
         1. [=map/Set=] |browserSignals|["`joinCount`"] to the sum of |ig|'s

```

Comment:
should I specify the type of every field of record's `any` value, even when it's quite obvious? I guess yes?

Reply:
defined a dictionary for bidding browser signals. Will do the same for other methods' browserSignals as well in separate PRs.